### DISCUSSION TITLE
Editing the data grid

### DISCUSSION THREAD

**Discussion by fluke030** [AvaloniaUI/Avalonia#14202] [Q&A] [answered]
Hello,
I followed the [example to use a data grid](https://docs.avaloniaui.net/docs/reference/controls/datagrid/), but mine is not editable. I am not sure what I have done wrong. I can see the data from the VM, but I cannot edit it.

The datagrid definition in my axaml form.

    <DataGrid Margin="20" 
            ItemsSource="{Binding People, Mode=TwoWay}"
            CanUserResizeColumns="True"
            CanUserSortColumns="True"
            GridLinesVisibility="All"
            IsReadOnly="False"
            BorderThickness="1" BorderBrush="Gray">
      <DataGrid.Columns>
        <DataGridTextColumn Header="First Name"  Binding="{Binding FirstName}"/>
        <DataGridTextColumn Header="Last Name" Binding="{Binding LastName}" />
        <DataGridCheckBoxColumn Header="Fictitious?" Binding="{Binding IsFictitious}" />
      </DataGrid.Columns>
    </DataGrid>

The view model.

    public class MainWindowViewModel : ViewModelBase
    {
        private ObservableCollection<Person> _people;

        public ObservableCollection<Person> People 
        {
            get => _people;
            set { this.RaiseAndSetIfChanged(ref _people, value); }
        }

        public MainWindowViewModel()
        {
            var people = new List<Person>
            {
                new Person("Neil", "Armstrong", false),
                new Person("Buzz", "Lightyear", true),
                new Person("James", "Kirk", true)
            };
            People = new ObservableCollection<Person>(people);
        }
    }

I have also created a [repo](https://gitlab.com/fluke030/avaloniadatagrid) with the example project which does not edit the datagrid.

**Comment by maxkatz6** [comment, marked as answer]
11.0.6? Please try 11.0.5 or latest nightly builds. There was an issue with DataGrid that should be fixed in the next stable release.

**Reply by fluke030** [reply]
Thank you! That did it!
